### PR TITLE
[Automation]Volume migration between pools times out in ACS, but  but the migration completes on Xenserve

### PR DESCRIPTION
**Pull request by nitt10prashant**
Ticket :CLOUDSTACK-8771

description:
Volume migration between pools times out in ACS, but the migration completes on Xenserver

test results
========
put storage in maintenance mode and start ha vm and check usage ... === TestName: test_migrate_volume_timeout | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 250.958s

OK

### CODE REVIEW

File: test/integration/component/maint/test_migrate_volume_timeout.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                    cls.find_storage_pool = False
+                    return
+            clusters = Cluster.list(cls.api_client, zoneid=cls.zone.id)
+
+            if not validateList(clusters)[0]:
+
+                cls.debug(
+                    "check list cluster response for zone id %s" %
+                    cls.zone.id)
+
```
Comment (sanju1010) [inline test/integration/component/maint/test_migrate_volume_timeout.py]: Why are we not failing the test if the list cluster response is not valid?

File: test/integration/component/maint/test_migrate_volume_timeout.py [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+                                            clusterid=cluster.id,
+                                            keyword="NetworkFilesystem"
+                                            )
+
+                if not validateList(cls.pool)[0]:
+
+                    cls.debug(
+                        "check list cluster response for zone id %s" %
+                        cls.zone.id)
+
```
Comment (sanju1010) [inline test/integration/component/maint/test_migrate_volume_timeout.py]: Same comment here!!

**Comment by bhaisaab** [comment]
I lack the infra to test this, but LGTM; but maybe fix that functional issue @sanju1010 has commented

**Comment by remibergsma** [comment]
@nitt10prashant Any update on this?

**Pull request closed without merging**